src/base/static/js/views/app-view.js

```diff
@@ -172,6 +173,8 @@ export default Backbone.View.extend({
     this.activities = this.options.activities;
     this.places = this.options.places;
 
+    this.fetchAndLoadDatasets();
+
```

Comment:
Just FYI, note that this is an async function, so this doesn't guarantee that datasets will be loaded at any point.

But I don't think there is a better alternative for now - hopefully this will be addressed when we dissolve AppView.